packages/server/src/membership/membership.entity.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  BaseEntity,
+  Entity,
+  PrimaryKey,
+  Property,
+  Unique,
+  Collection,
+  OneToMany,
+  OneToOne,
+} from '@mikro-orm/core';
+import { User } from '../user/user.entity';
+import { Invoice } from '@server/invoice/invoice.entity';
+
+@Entity()
+//@Unique({ properties: ['user'] })
+export class Membership extends BaseEntity<Membership, 'id'> {
+  @PrimaryKey()
+  id!: number;
+
+  @OneToOne(() => User, (user) => user.membership)
+  user!: User;
```

Comment:
I wonder if perhaps this should be a `ManyToOne` thinking when the membership expires. Once a user needs to pay to re-establish their membership, overwriting the last one erases some information. Otherwise, more careful invoice work could be considered.